lib/cppcheck.cpp

```diff
@@ -432,6 +432,9 @@ static std::vector<picojson::value> executeAddon(const AddonInfo &addonInfo,
                                                  const std::string &premiumArgs,
                                                  const CppCheck::ExecuteCmdFn &executeCommand)
 {
+    if (!executeCommand)
+        throw InternalError(nullptr, "Failed to execute addon - no command callback provided");
```

Comment:
hmm.. ideally I would like to make it clear that it's a bug in cppcheck. The user should not start looking for some command callback options. but I don't have a very good suggestion..

Reply:
Well - it should never happen. It's kind of akin to "unreachable" code.

Reply:
yes but I think this `throw` is valuable, we could have another issue again 10 years from now..

Reply:
I resolve because I don't have a very good suggestion..